src/components/Exchange/PositionSeller.js

```diff
@@ -835,6 +836,10 @@ export default function PositionSeller(props) {
 
   const shouldShowExistingOrderWarning = false;
 
+  if (!isSwapAllowed && !position.isLong && !triggerPriceUsd) {
```

Comment:
I think it would make more sense to use the orderOption === STOP condition here instead of isSwapAllowed

Because isSwapAllowed is not related to triggerPrice and it's not really clear why this condition is here until you dive into the code

Reply:
Done!